cmd/tarianctl/cmd/version.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Args:  cobra.NoArgs,
+	Short: "Prints version of tarianctl",
+	Run: func(cmd *cobra.Command, args []string) {
+		// need to implement this
```

Comment:
In existing version, it's already implemented and straightforward to copy it to Cobra.

https://github.com/kube-tarian/tarian/blob/main/cmd/tarianctl/tarianctl.go#L15-L19

These variables are substituted by goreleaser with the respected release version (snapshot, tag, etc).

```go
// nolint: gochecknoglobals
var (
	version = "dev"
	commit  = "main"
)
```

Then the version string looks like this:

```go
versionStr := version + " (" + commit + ")",
```

Reply:
Ah, got it. Earlier I wasn't sure how the versioning is working here. So I did not implement it.